notification/test/integration/notification.handler.spec.js

```diff
@@ -30,11 +30,12 @@ describe('notification module', () => {
     await iTSetUp.cleanupProject(ctpClient)
   })
 
-  it('should update the transaction state when receives a correct notification', async () => {
+  it('should update the pending authorization transaction state to success state ' +
+    'when receives a successful AUTHORIZATION notification', async () => {
     const { body: { results: [ paymentBefore ] } } = await ctpClient.fetch(ctpClient.builder.payments)
     expect(paymentBefore.transactions).to.have.lengthOf(1)
     expect(paymentBefore.transactions[0].type).to.equal('Authorization')
-    expect(paymentBefore.transactions[0].state).to.equal('Initial')
```

Comment:
So there will be no transaction in `Initial` state?

Reply:
according to our documentation, it's better to use pending in this test, it's pending state because the payment will go first to the extension module, so notification could trigger after some operations which are done by extension module, right?